### PR TITLE
Add Kubemark Benchmark Design Doc

### PR DESCRIPTION
**Pull request by Rajadeepan**
**What this PR does / why we need it**:
   This PR illustrates the Design Doc for kubemark Benchmark in Kube-Batch

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
https://github.com/kubernetes-sigs/kube-batch/issues/427

**Special notes for your reviewer**:
  https://github.com/Rajadeepan/kube-batch/blob/doc/doc/design/Benchmark/kubemark/kubemark-benchmarking.md

### CODE REVIEW

File: doc/design/Benchmark/kubemark/kubemark-benchmarking.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## BACKGROUND
+
+Performance comparison of KubeBatch scheduler with default scheduler. Kubemark will be used as a tool for benchmarking.
+Kubemark is a performance testing tool which we use to run simulated kubernetes clusters. The primary use case is scalability 
+testing, as creating simulated clusters is faster and requires less resources than creating real ones.
+
+## OBJECTIVE
+
+- Comparing the scheduler performance of kube-batch with the scheduler performance of the default scheduler
```
Comment (k82cn) [inline doc/design/Benchmark/kubemark/kubemark-benchmarking.md]: Our objective is to evaulate performance of kube-batch; it's not necessary to compare with default scheduler.

File: doc/design/Benchmark/kubemark/kubemark-benchmarking.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## DESIGN OVERVIEW
+
+We assume that we want to benchmark a test T across two variants A(with default scheduler) and B(with kube-batch scheduler).
```
Comment (k82cn) [inline doc/design/Benchmark/kubemark/kubemark-benchmarking.md]: Nop, only evaulate kube-batch's performance is ok.

**Comment by Rajadeepan** [comment]
@k82cn updated the document as per your review comment. Does this benchmarking strategy looks fine? can i go ahead and get the benchmarking results using this strategy?

**Pull request merged**